src/components/App.js

```diff
@@ -197,6 +198,11 @@ class App extends Component {
                     component={WorkOrderDetails}
                     isAuthenticated={this.state.knackUserToken}
                   />
+                  <PrivateRoute
+                    path="/assets/"
```

Comment:
I think the Assets search page is a nice feature, worth keeping. But we need to also implement the routing for a path like `path="/assets/:assetId"`. That way we can link from a Work Order to an asset details page where the big Location Name header on the work order details page links to the individual asset. I think that answers your third question.

Reply:
Awesome, sounds like a plan. I'll add the `/assets/:assetId` route and disable the search bar when used. I'm limiting the link on the Work Order Details view to Signals for now because the Knack app does not have an asset link for School Beacons (https://atd.knack.com/7-feb-2019--test-atd-data-tracker#work-orders/work-order-details/5d01787f922972000da0a5d8/). I need to investigate further to see what is present on the Knack side for other asset types.